packages/web/src/components/range/RangeSlider.js

```diff
@@ -296,7 +434,16 @@ class RangeSlider extends Component {
 				size: 0,
 				aggs: (props.histogramQuery || this.histogramQuery)(props),
 			};
-			const value = [props.range.start, props.range.end];
+			const value = [
+				getNumericRangeValue(
```

Comment:
@mohdashraf010897 Can you create a `getter` method to get the range values? You're repeating this logic several times. Applies to DRS as well.

Reply:
@bietkul I have done the changes for RangeSlider, for DRS I didn't find the need to since the range value is always numeric that we get from the backend.